### DISCUSSION TITLE
Development version and translation

### DISCUSSION THREAD

**Discussion by Denebet** [quodlibet/quodlibet#4539] [Q&A]
Is it possible to use a development version with a translation, without building and installing QL?

**Comment by Denebet** [comment]
Both for my comfort and to test my improvements before publishing them...

**Comment by declension** [comment]
AFAIK I'm afraid not. The git version doesn't store compiled translations, just their source. So you'd have to do the build.

(Other than dev PPAs etc which are not that supported any more)

**Comment by Denebet** [comment]
No problem compiling the .po into .mo, but what do I do with it afterwards?
The Quod Libet website, at:
https://quodlibet.readthedocs.io/en/quodlibet-4.6/development/overview.html
says:
"While Quod Libet uses distutils/setup.py for building and installing (./setup.py build install etc.) we don’t recommend it, as it doesn’t provide a way to uninstall the application again and it might not do the right thing by default depending on your distribution/operating system."
If I take the risk of compiling, can you indicate the procedure to uninstall (completely clean up) afterwards, if necessary?
